app/controllers/graphql_metrics/dashboard_controller.rb

```diff
@@ -1,4 +1,5 @@
 require_dependency "graphql_metrics/application_controller"
+require "graphql_metrics/version"
```

Comment:
I needed to add this - because I was getting a constant missing error for GraphqlMetrics::VERSION